### PR TITLE
aggregator update/create version tests.

### PR DESCRIPTION
**Pull request by niedbalski**
# Summary of this proposal

Extend update/create aggregator tests to include the recently added version field.

## Notes for the reviewer

N/A

## Issue(s) number(s)

N/A

## Checklist for submitter

- [X] My PR provides tests
  - [ ] Integration tests are added/passes
  - [X] Unit tests are added/passes
  - [X] End-to-end tests added/passes
  - [X] Static code check added/passes
  - [X] Linting on documentation added/passes
  - [ ] Does not affect code coverage stats

## Checklist for reviewer

- [ ] The proposal fixes a bug/issue or implements a new feature that is well described.
- [ ] The proposal has sufficient test cases that covers the changes.
  - [ ] If changes an API, it does not break backwards compatibility (-1 major version).
- [ ] If integration is required, the proposal has integration tests.
- [ ] The proposal does not break coverage stats
- [ ] The proposal has the required documentation changes.

## Backport

- [ ] Backport to the latest stable release.

### CODE REVIEW

File: client/aggregator_test.go [hunk 1]
```diff
@@ -15,16 +15,34 @@ func TestClient_CreateAggregator(t *testing.T) {
...
+		})
+
+		wantEqual(t, err, nil)
+
+		got, err = withToken.CreateAggregator(ctx, types.CreateAggregator{
+			Name: "duplicate",
+		})
+
+		wantEqual(t, got, types.CreatedAggregator{})
+		wantNoEqual(t, err, nil)
```
Comment (nicolasparada) [inline client/aggregator_test.go:44]: Maybe you could add: 
```go
wantErrMsg(t, err, "aggregator name taken")`
```

```go
func wantErrMsg(t *testing.T, err error, msg string) {
  t.Helper()
  if err == nil || (err != nil && !strings.Contains(err.Error(), msg)) {
    t.Fatalf("want msg %q; got %v", msg, err)
  }
}
```

Comment (niedbalski) [reply client/aggregator_test.go]: done

File: client/aggregator_test.go [hunk 2]
```diff
@@ -2,29 +2,55 @@ package client_test
...
 import (
 	"context"
+	"strings"
 	"testing"
 
 	"github.com/calyptia/api/client"
 	"github.com/calyptia/api/types"
 )
 
+func wantErrMsg(t *testing.T, err error, msg string) {
```
Comment (nicolasparada) [inline client/aggregator_test.go:12]: Can you move this to `client_test.go` file?
Similar utilities are already there.

File: client/aggregator_test.go [hunk 3]
```diff
@@ -2,29 +2,55 @@ package client_test
...
 	wantEqual(t, len(got.ResourceProfiles), 3)
+
+	t.Run("name exists", func(t *testing.T) {
+		_, err := withToken.CreateAggregator(ctx, types.CreateAggregator{
+			Name: "duplicate",
+		})
+
+		wantEqual(t, err, nil)
+
+		got, err = withToken.CreateAggregator(ctx, types.CreateAggregator{
```
Comment (nicolasparada) [inline client/aggregator_test.go:47]:
```suggestion
		_, err = withToken.CreateAggregator(ctx, types.CreateAggregator{
```

File: client/aggregator_test.go [hunk 4]
```diff
@@ -2,29 +2,55 @@ package client_test
...
+			Name: "duplicate",
+		})
+
+		wantEqual(t, got, types.CreatedAggregator{})
```
Comment (nicolasparada) [inline client/aggregator_test.go:51]:
```suggestion
```
There is no need for that. We are asserting the error.

**Review by nicolasparada — APPROVED** [review]
Nice

**Pull request merged**